test/Infrastructure.IntegrationTest/AdminConsole/Repositories/OrganizationDomainRepositoryTests.cs

```diff
@@ -0,0 +1,136 @@
+﻿
+using Bit.Core.Repositories;
+using Bit.Infrastructure.EntityFramework.AdminConsole.Models;
+using Bit.Infrastructure.EntityFramework.Models;
+using Xunit;
+
+namespace Bit.Infrastructure.IntegrationTest.Repositories;
+
+public class OrganizationDomainRepositoryTests
+{
+    [DatabaseTheory, DatabaseData]
+    public async Task GetExpiredOrganizationDomainsAsync_ShouldReturnExpiredOrganizationDomains(
```

Comment:
hey @eliykat 

I’m trying to add a test for the VerifiedDate field, and I need a way to update it. My plan is to create a class that extends OrganizationDomain and add test logic to update the VerifiedDate, since this will only be used in test env. 

Just wanted to check if this is an acceptable pattern here or if there’s a better way to handle this.

Reply:
Which specific test are you trying to add? It looks like you're already using the `.SetVerifiedDate()` method here, and it seems to be fulfilling the purpose for these tests.

Reply:
Maybe you meant `CreationDate`? Well, that one seems tricky. Maybe you can use `ITimeProvider` to set the value and then mock it on the test?

Reply:
Sorry, I meant `CreationDate`. Thanks, I'll look into `ITimeProvider`.

Reply:
After a nice weekend of rest, this is actually easier than I thought, lol. I added the new tests.